tests/onlineboutique_with_replicas_and_variants/pods.yaml

```diff
@@ -955,7 +955,6 @@ items:
     labels:
       app: loadgenerator
       pod-template-hash: 555fbdc87d
-      unhealthy: "true"
```

Comment:
Please check `eval_test.go` lines 1653-1661. The test specifically specifies "loadgenerator pod has 3 replicas but one with variant on label" , and this is required for the test. Doesn't your change break this test?

Reply:
my changes, cause the cache hits count to be raised (was 25 now 49, i.e before 2 replicas were counted now 3)
this does not break the test , since the `expectedCacheHits` is not const , it is computed dynamically through the test run (in line 1677) 
so both the actual and expected values are changed and equal in both cases

Reply:
however since currently the changes affect only connectivity analysis commands and not eval command. (the check of labels per owner ref. is moved to `getpeersList func)` , and since this test is used only in `eval_test.go` , 
**reverted this change**

Reply:
Now that this change was reverted it's ok. (before revert, the change did not break the test, but had made it the same as the test above it , and thus the functionality required for the test was no longer tested... there was intentionally one test that has a pod with 3 replicas, and another test that has pod with 3 replicas but one with variant on the labels).